cmd/pj-rehearse/main.go

```diff
@@ -332,18 +344,21 @@ func rehearseMain() int {
 	}
 	if err := prConfig.Prow.ValidateJobConfig(); err != nil {
 		logger.WithError(err).Error("jobconfig validation failed")
-		return 1
+		fmt.Fprintf(os.Stderr, jobValidationOutput)
+		return FAILURE
 	}
 
 	executor := rehearse.NewExecutor(presubmitsToRehearse, prNumber, o.releaseRepoPath, jobSpec.Refs, o.dryRun, loggers, pjclient)
 	success, err := executor.ExecuteJobs()
 	if err != nil {
 		logger.WithError(err).Error("Failed to rehearse jobs")
-		return gracefulExit(o.noFail, rehearseFailureOutput)
+		fmt.Fprintf(os.Stderr, rehearseFailureOutput)
+		return FAILURE
 	}
 	if !success {
 		logger.Error("Some jobs failed their rehearsal runs")
-		return gracefulExit(o.noFail, jobsFailureOutput)
+		fmt.Fprintf(os.Stderr, jobsFailureOutput)
```

Comment:
This doesn't follow our normal convention. Why not return an `error` and, if that's not `nil`, have the caller print that to `os.Stderr` and exit non-zero?

Reply:
Well, I just inlined what was already there /shrug. But I can do it this way, no problem.

Reply:
Addressed.